spec/features/create_new_collection_spec.rb

```diff
@@ -0,0 +1,32 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Create a new collection', js: true do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in user, groups: ['dlss:hydrus-app-collection-creators']
+    allow(Settings).to receive(:allow_sdr_content_changes).and_return(true)
```

Comment:
I suspect this is not needed here as it's the default value and this test isn't testing the `false` case.